config/formats.js

```diff
@@ -707,6 +674,37 @@ let Formats = [
 		team: 'randomHC',
 		searchShow: false,
 		ruleset: ['Pokemon', 'HP Percentage Mod', 'Cancel Mod'],
+	},	{
```

Comment:
?

Reply:
what austin means is that all the other are like 
```javascript
},
{
```

Reply:
Sorry I assumed that was incredibly obvious